template_config.yml

```diff
@@ -1,30 +1,33 @@
 # This config represents the latest values used when running the plugin-template. Any settings that
 # were not present before running plugin-template have been added with their default values.
 
-plugin_name: pulp-2to3-migration
+additional_plugins: []
```

Comment:
this plugin needs `pulp_file` and `pulp_docker`:
```yaml
additional_plugins:
  - pulp_file
  - pulp_docker
```
after add it, it is necessary to re-run:
```
./plugin-template --travis pulp-2to3-migration
```
But I've opened a PR also: https://github.com/pulp/pulp-2to3-migration/pull/33

Reply:
@fabricio-aguiar yeah i just noticed that so i will close mine, thanks for reacting so fast in fixing issues!